Xming: Reduce MOUSE_POLLING_INTERVAL

Reduce MOUSE_POLLING_INTERVAL from 500ms to 50ms

In -multiwindow mode, this determines how quickly X windows can track the mouse
when it is outside any X window... Move the cursor around an xeyes, and you will
see what this timing effects.

---
 xserver/hw/xwin/win.h                     |    1 +
 xserver/hw/xwin/winmultiwindowwindow.c    |    7 -------
 xserver/hw/xwin/winmultiwindowwndproc.c   |    1 -
 xserver/hw/xwin/winwin32rootless.c        |    3 ---
 xserver/hw/xwin/winwin32rootlesswndproc.c |    1 -
 5 files changed, 1 insertion(+), 12 deletions(-)

Index: xorg-server-1.5.3/xserver/hw/xwin/win.h
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/win.h	2009-01-07 21:02:00.000000000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/win.h	2009-01-07 21:34:02.765625000 +0000
@@ -103,6 +103,7 @@
 #define WIN_E3B_TIMER_ID		1
 #define WIN_POLLING_MOUSE_TIMER_ID	2
 
+#define MOUSE_POLLING_INTERVAL		50
 
 #define WIN_E3B_OFF		-1
 #define WIN_FD_INVALID		-1
Index: xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwindow.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winmultiwindowwindow.c	2009-01-07 21:01:38.000000000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwindow.c	2009-01-07 21:08:18.796875000 +0000
@@ -65,13 +65,6 @@
 winFindWindow (pointer value, XID id, pointer cdata);
 
 /*
- * Constant defines
- */
-
-#define MOUSE_POLLING_INTERVAL		500
-
-
-/*
  * Macros
  */
 
Index: xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwndproc.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winmultiwindowwndproc.c	2009-01-06 20:46:30.000000000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwndproc.c	2009-01-07 21:09:18.125000000 +0000
@@ -65,7 +65,6 @@
  * Constant defines
  */
 
-#define MOUSE_POLLING_INTERVAL		500
 #define WIN_MULTIWINDOW_SHAPE		YES
 
 
Index: xorg-server-1.5.3/xserver/hw/xwin/winwin32rootless.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winwin32rootless.c	2009-01-07 21:07:17.000000000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winwin32rootless.c	2009-01-07 21:07:50.843750000 +0000
@@ -48,9 +48,6 @@
 /*
  * Constant defines
  */
-
-#define MOUSE_POLLING_INTERVAL		500
-
 #define ULW_COLORKEY	0x00000001
 #define ULW_ALPHA	0x00000002
 #define ULW_OPAQUE	0x00000004
Index: xorg-server-1.5.3/xserver/hw/xwin/winwin32rootlesswndproc.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winwin32rootlesswndproc.c	2009-01-07 21:01:16.000000000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winwin32rootlesswndproc.c	2009-01-07 21:09:54.203125000 +0000
@@ -48,7 +48,6 @@
  * Constant defines
  */
 
-#define MOUSE_POLLING_INTERVAL		500
 #define MOUSE_ACTIVATE_DEFAULT		TRUE
 #define RAISE_ON_CLICK_DEFAULT		FALSE
 
